.prettierrc.json

```diff
@@ -1 +1,7 @@
-{}
+{
+  "printWidth": 120,
+  "singleQuote": true,
+  "tabWidth": 4,
```

Comment:
❓ Isn't `tabWidth` redundant if we have `"useTabs": true`? `tabWidth` controls the number of spaces per "tab", but that's irrelevant if we don't use spaces at all.

Reply:
Copy/pasted from Louis, but I think you're right. Removing it.